packages/app/src/cli/services/dev/extension/server/utilities.test.ts

```diff
@@ -124,7 +133,27 @@ describe('getExtensionPointRedirectUrl()', () => {
     )
   })
 
-  test('returns customer account server URL if the extension point uses legacy customer account target', () => {
+  test('returns customer account URL on a spin shopify domain if it is in a spin env', () => {
+    vi.mocked(isSpinEnvironment).mockReturnValue(true)
+    const extension = {
+      devUUID: '123abc',
+    } as ExtensionInstance
+
+    const options = {
+      storeFqdn: 'example.myshopify.com',
```

Comment:
could be interesting using a formatted spin store url here like `shop1.shopify.spin-instance.eu.spin.dev`

Reply:
hey @lihaokx . This error

```
Error: Request failed due to following response errors:
 - Although you appear to have the correct authorization credentials, the `Shopify` organization has an IP allow list enabled, and your IP address is not permitted to access this resource.
```
is totally unrelated. If you run into it you just need to retry until it works :sweat_smile: